docs/sources/guides/whats-new-in-v6-5.md

```diff
@@ -0,0 +1,94 @@
+- [**Docker:** Ubuntu-based images and more]({{< relref "#ubuntu-based-docker-images" >}})
+- **CloudWatch:** Use GetMetricData API instead of GetMetricStatistics API
+- **CloudWatch:** Dynamic queries using wildcards for dimension values
+- **CloudWatch:** Deep linking from Grafana panels to the CloudWatch console
+- **CloudWatch:** Improved feedback when throttling occurs
+- [**Templating:** Dynamic typeahead queries using $__searchFilter]({{< relref "#dynamic-typeahead-support-in-query-variables" >}})
+- [**Explore:** New log row details view]({{< relref "#explore-logs-log-row-details" >}})
+- [**Explore:** Turn parts of log message into a link using derived fields]({{< relref "#loki-explore-derived-fields" >}})
+- [**Explore:** Time-sync of split views]({{< relref "#time-sync-of-split-views-in-explore" >}})
+- **Explore**: Tooltip in graphs
+- **Azure Monitor**: Alerting support for Azure Application Insights
+- **Provisioning**: Allow saving of provisioned dashboards from UI
+- **Auth Proxy:** Can now login with auth proxy and get a login token and session cookie
+- **OAuth:** Generic OAuth now supports role mapping
+
+More details of above and highlights will be added as we're getting closer to the stable release.
+
+### Ubuntu-based docker images
+
+In Grafana [v6.4](/guides/whats-new-in-v6-4/#alpine-based-docker-image) we switched the Grafana docker image from Ubuntu to Alpine. The main reason for this change was to be able to provide a more secure and lightweight docker image.
+
+This change has received both negative and positive feedback as well as some bug reports. Based on this, one of the conclusions and learnings is that switching to an Alpine based docker image was a big breaking change for a lot of users and this change should have been more clearly highlighted in blog post, release notes, changelog and the [Docker Hub readme](https://hub.docker.com/r/grafana/grafana).
+
+One additional mistake we did was to break the Docker images for ARM. Good news, in Grafana v6.5 this have been fixed.
+
+Grafana docker images should be as secure as possible by default and that’s why the Alpine based docker images will continue to be provided as Grafana’s default (`grafana/grafana:<version>`).  With that said, it’s good to give users options and that’s why starting from Grafana v6.5 there’re also Ubuntu based docker images (`grafana/grafana:<version>-ubuntu`) available.
+
+### Dynamic typeahead support in query variables
+
+If you have a query variable that has many thousands of values it can be quite slow to search for a specific value in the dropdown. This is due to the fact that all that search filtering is happening in the browser.
+
+Using `__searchFilter` in the template variable query field you can filter the query results based on what the user types in the variable dropdown input.
+
+When nothing has been entered by the user the default value for `__searchFilter` is `*` ,  `.*` or  ‘%’  depending on data source and formatting option.
```

Comment:
Is there a reason for different quoting of %?

Reply:
Thanks. Fixed and pushed